pkg/model/resource/webhooks.go

```diff
@@ -77,10 +101,36 @@ func (webhooks *Webhooks) Update(other *Webhooks) error {
 	// Update conversion.
 	webhooks.Conversion = webhooks.Conversion || other.Conversion
 
+	// Update Spoke (merge without duplicates)
+	if len(other.Spoke) > 0 {
```

Comment:
Is there a way to find if the provided spoke version is not one of hub? Given we are not storing the hub version in `Webhooks` struct? Or are we catching that error before?

Reply:
Sorry, just tested this out. Looks like we are catching it. Pls ignore.